src/RockPaperScissors/RpsServer/Controllers/RpsController.cs

```diff
@@ -0,0 +1,210 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using RpsServer.Models;
+
+namespace RpsServer.Controllers
+{
+    [Route("api/[controller]")]
+    public class RpsController : Controller
+    {
+        private readonly GameContext context;
+
+        public RpsController(GameContext context)
+        {
+            this.context = context;
+        }
+
+        #region APIs
+        [HttpGet]
```

Comment:
I think most of my commands _should_ be POST. But GET has a slight advantage for testing, since you can easily through any of the api calls into a browse.

I do a lot of Get/Create operations here, which begs the question, should these all be POST (or perhaps PUT in some cases), or should we separate the two into different commands. As in you, the client, could call GET to see if it exists, then call POST if it doesn't. I think this is technically the more restful approach, but it also adds complexity to the client, and increases the number of network connections.